src/controllers/UserController.js

```diff
@@ -219,6 +219,40 @@ class UserController {
       return HelperMethods.serverError(res);
     }
   }
+
+  /**
+  * Sends Emails To Users For Password Reset.
+  * Route: POST: /api/v1/resetpassword
+  * @param {object} req - HTTP Request object
+  * @param {object} res - HTTP Response object
+  * @param {object} next - HTTP Response object
+  * @return {res} res - HTTP Response object
+  * @memberof UserController
+ */
+  static async resetPassword(req, res) {
+    try {
+      const { email } = req.body;
+      const userFound = await User.findOne({ where: { email } });
+      if (!userFound) {
+        return HelperMethods.clientError(res, {
+          success: false,
+          message: 'Email does not exist',
+        }, 400);
+      }
+      if (userFound) {
+        const emailSent = await SendEmail.resetPassword(email);
+        if (emailSent) {
+          return HelperMethods.requestSuccessful(res, {
+            success: true,
+            message: 'An email has been sent to you '
+              + 'check your email address'
+          }, 200);
```

Comment:
Please make your response message more descriptive. `An email has been sent to your email address that explains how to reset your password.`